gobblin-utility/src/test/java/gobblin/util/io/GsonInterfaceAdapterTest.java

```diff
@@ -0,0 +1,29 @@
+package gobblin.util.io;
```

Comment:
I added the license headers. Don't quite understand what you mean by `@Test groups`.